src/boot/mlang.ml

```diff
@@ -159,16 +159,16 @@ let translate_cases f target cases =
   let translate_case case inner =
     match case with
     | (ConPattern (fi, k, x), handler) ->
-      TmMatch (fi, target, PatCon(fi, k, noidx, PatNamed(fi, NameStr(x))), handler, inner)
+      TmMatch (fi, target, PatCon(fi, k, noidx, PatNamed(fi, NameStr(x,0))), handler, inner)
```

Comment:
I feel like `0` here should be `noidx`, no? Probably won't matter since `symbolize` is run after, but still.

Reply:
Changed to `nosym` insteadof `noidx`and updated on the necessary places in `mlang.ml`.